ReactAndroid/src/main/java/com/facebook/react/bridge/ReadableNativeArray.java

```diff
@@ -46,4 +49,40 @@ protected ReadableNativeArray(HybridData hybridData) {
   public native ReadableNativeMap getMap(int index);
   @Override
   public native ReadableType getType(int index);
+
+  @DoNotStripe
```

Comment:
Not yet. Will update the description that it's pending and resolve that note once confirmed.